modules/network-graph-20-view.adoc

```diff
@@ -51,24 +76,22 @@ If you are using offline mode, you can update these ranges by installing new sup
 
 The following image provides an example of the network graph. In this example, based on the options that the user has chosen, the graph depicts deployments in the selected namespace. Traffic flows are not displayed until you click on an item such as a deployment. The graph uses a red badge to indicate deployments that are missing policies and therefore allowing all network traffic.
 
-.Network graph example
-image::network-graph-20-overview-updated.png[Overview of the Network graph showing deployments]
+[id="internal-entities-connections_{context}"]
+== Connections involving internal entities
 
-When you click an item in the graph, the rearranged side panel with collapsible sections presents information about that item. You can click on the following items:
+The network graph is useful for identifying deployments with active connections to entities that do not belong to any known deployment or CIDR block. Some of these connections never reach outside of the cluster and are made within the cluster's private network. The network graph represents those as connections to or from _internal entities_.
 
-* Deployments
-* Namespaces
-* External entities
-* CIDR blocks
-* External groups
+Connections with internal entities represent a connection between a deployment and an IP address that belongs to the private address space as defined in http://datatracker.ietf.org/doc/html/rfc1918[RFC 1918]. In some cases, Sensor is unable to identify one or both deployments involved in a connection. In that case, the system analyzes the IP address and decides whether the connection is internal or external.
 
-The side panel displays relevant information based on the item in the graph that you have selected. The *D* or *NS* label next to the item name in the header (in this example, "visa-processor,") indicates if it is a deployment or a namespace. The following example illustrates the side panel for a deployment.
+The following scenarios can lead to a connection being categorized as one involving internal entities:
 
-.Side panel for a deployment example
-image::network-graph-20-deployment-mode-updated.png[Side panel for a deployment]
+* A change of IP address or the deletion of a deployment accepting connections (the server) while the party initiating the connection (the client) still attempts to reach it
+* A deployment communicating with the orchestrator API
+* A deployment communicating using a networking CNI plugin, for example, Calico
+* A restart of Sensor, resulting in a reset of the mapping of IP addresses to past deployments; for example, when Sensor does not recognize the IP addresses of past entities or past IP addresses of existing entities
```

Comment:
Per ISG, when the text after "for example" isn't an independent clause, then you don't use a semicolon. Just a comma.

https://www.ibm.com/docs/en/ibm-style?topic=format-examples#inline-examples

```suggestion
* A restart of Sensor, resulting in a reset of the mapping of IP addresses to past deployments, for example, when Sensor does not recognize the IP addresses of past entities or past IP addresses of existing entities
```